modules/extensions-support/src/main/java/org/mule/module/extension/internal/util/IntrospectionUtils.java

```diff
@@ -280,8 +281,8 @@ public static boolean isRequired(ParameterModel parameterModel, boolean forceOpt
 
     public static ExpressionSupport getExpressionSupport(AccessibleObject object)
     {
-        Parameter parameter = object.getAnnotation(Parameter.class);
-        return parameter != null ? parameter.expressionSupport() : SUPPORTED;
+        Expression expression = object.getAnnotation(Expression.class);
```

Comment:
this is the second time I read these three lines. Create a static method in IntrospectionUtils which do this comparisson and centralize the default value